### PR TITLE
Explicit mapping for all osquery fields

### PR DESCRIPTION
**Pull request by aleksmaus**
## What does this PR do?

Explicit mapping for all osquery fields.

Following this recommendation for mapping
https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html

```
Consider mapping a numeric identifier as a keyword if:
You don’t plan to search for the identifier data using range queries.
Fast retrieval is important. term query searches on keyword fields are often faster than term searches on numeric fields.
If you’re unsure which to use, you can use a multi-field to map the data as both a keyword and a numeric data type.
```

The fields mapping is generated from the official osquery 4.7.0 schema. The numeric fields are mapped to keywords, the text fields had additional text multifield defined.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.

## Related issues
- Relates https://github.com/elastic/security-team/issues/935

## Screenshots

osquery mapping:
Screen Shot 2021-04-10 at 4 27 22 PM

collected osquery data with types converted appropriately with osquerybeat
Screen Shot 2021-04-10 at 4 25 11 PM

### CODE REVIEW

**Comment by aleksmaus** [comment]
Do not review/merge yet, will update the mapping more today.

File: packages/osquery_manager/manifest.yml [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+version: 0.2.0
 license: basic
 description: Osquery Manager Integration
 type: integration
 release: experimental
+categories:
+  - security
+  - os_system
 conditions:
   kibana.version: '^7.9.0'
```
Comment (patrykkopycinski) [inline packages/osquery_manager/manifest.yml:13]: @aleksmaus can we increase that to `7.13.0`?

Comment (aleksmaus) [reply packages/osquery_manager/manifest.yml]: done

Comment (patrykkopycinski) [reply packages/osquery_manager/manifest.yml]: 🙏

**Comment by aleksmaus** [comment]
the integration and kibana are updated to handle keyword/long keyword/double multifields correctly

Screen Shot 2021-04-13 at 11 40 37 AM
Screen Shot 2021-04-13 at 11 40 25 AM


Screen Shot 2021-04-13 at 11 42 15 AM

**Review by lykkin — APPROVED** [review]
out of curiosity, what did you use to generate the osquery.yml file?

**Review by james-elastic — APPROVED** [review]

**Pull request merged**